lib/workers/repository/process/index.ts

```diff
@@ -57,6 +57,12 @@ async function getBaseBranchConfig(
     }
 
     baseBranchConfig = mergeChildConfig(config, baseBranchConfig);
+
```

Comment:
```suggestion

logger.debug({ config: baseBranchConfig }, 'Base branch config prior to merge');

```

Reply:
@rarkins are you sure you want to print it all without the printConfig=true?? it prints the whole config!